tools/ci-cdk/canary-runner/src/run.rs

```diff
@@ -83,6 +83,11 @@ pub struct RunArgs {
     #[clap(long)]
     musl: bool,
 
+    /// Expected speech text generated by Transcribe. This needs to be passed-in
+    /// because it can change as the accuracy of generated text improves over time.
+    #[clap(long)]
+    expected_speech_text_by_transcribe: String,
```

Comment:
I think we should still make this optional just so that it's still easy to run in local dev.

Reply:
That is useful for local development. Addressed in 4073c70. If `expected_speech_text_by_transcribe` is unspecified, the updated code does not set the environment variable `CANARY_EXPECTED_TRANSCRIBE_RESULT`. If it needs to be set, the argument must be passed-in.